src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorDiagnosticsEndpoint.cs

```diff
@@ -7,12 +7,16 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
```

Comment:
PSA; a [split diff view](https://github.com/dotnet/aspnetcore-tooling/pull/2973/files?diff=split) seems to be more readable for this particular PR.